compiler-core/src/language_server/completer.rs

```diff
@@ -25,6 +25,29 @@ use super::{
     compiler::LspProjectCompiler, files::FileSystemProxy, DownloadDependencies, MakeLocker,
 };
 
+#[derive(Copy, Clone)]
+enum SortPriority {
```

Comment:
This is called "sort priority", but it's not a sort priority, it's a description of the item which is used to determine the priority. CompletionKind or something perhaps?

Reply:
Yeah fair enough, I changed it to CompletionKindPriority since the priority is part of the type definition itself